modules/ocf_www/manifests/site/www.pp

```diff
@@ -31,6 +31,8 @@
       ensure => 'directory';
     '/var/www/html/.well-known/matrix/server':
       source => 'puppet:///modules/ocf_www/matrix-server';
+    '/var/www/html/.well-known/matrix/client':
```

Comment:
The docs say you should add a CORS header for browser clients:

> To work in browser based clients, the file must be served with the appropriate Cross-Origin Resource Sharing (CORS) headers. A recommended value would be Access-Control-Allow-Origin: * which would allow all browser based clients to view it.

Reply:
Thanks for pointing that out, forgot about that part. I'll add that right now.

Reply:
I'm not too familiar with puppet, do you know of a way to set a header for a single file or directory that is aliased? The only conditions I can see in the [Apache docs](https://httpd.apache.org/docs/current/mod/mod_headers.html#header) is `always` or `onsuccess`, and I'd be concerned with adding allow origin from anywhere to the entire www.

Reply:
If we changed the whole `/.well-known/matrix` path to be a rewrite to `matrix.obe` the headers from there would be passed through right? We already serve the client well-known file from there and could just add the server file.